src/snowflake/connector/__init__.py

```diff
@@ -47,6 +49,7 @@
 logging.getLogger(__name__).addHandler(NullHandler())
 
 
+@wraps(SnowflakeConnection.__init__)
```

Comment:
For my education, what is this for?

Reply:
It copies docstrings and other things to the annotated function https://docs.python.org/3/library/functools.html#functools.wraps